### ISSUE TITLE
Undefined properties in com_tags tags feed

### ISSUE THREAD

**Issue by Denitz** [joomla/joomla-cms#16951]
### Steps to reproduce the issue
Ensure that you have a tag.
Enable error logging in PHP settings.
Load tags feed via `/index.php?option=com_tags&view=tags&format=feed`

### Expected result
Feed XML

### Actual result
Feed XML

But, see notices in PHP error log (for some reason they are  not displayed even if you set Joomla error reporting to development level)
```
Notices:
[03-Jul-2017 14:59:41 UTC] PHP Notice:  Undefined property: stdClass::$author in /components/com_tags/views/tags/view.feed.php on line 54
[03-Jul-2017 14:59:41 UTC] PHP Notice:  Undefined property: stdClass::$displayDate in /components/com_tags/views/tags/view.feed.php on line 55
[03-Jul-2017 14:59:41 UTC] PHP Notice:  Undefined property: stdClass::$author_email in /components/com_tags/views/tags/view.feed.php on line 72
```

### System information (as much as possible)
PHP 7

**Comment by franz-wohlkoenig** [comment]
Confirmed:
```
[03-Jul-2017 18:50:01 Europe/Berlin] PHP Notice:  Undefined property: stdClass::$author in /Applications/MAMP/htdocs/3/components/com_tags/views/tags/view.feed.php on line 54
[03-Jul-2017 18:50:01 Europe/Berlin] PHP Notice:  Undefined property: stdClass::$displayDate in /Applications/MAMP/htdocs/3/components/com_tags/views/tags/view.feed.php on line 55
[03-Jul-2017 18:50:01 Europe/Berlin] PHP Notice:  Undefined property: stdClass::$author in /Applications/MAMP/htdocs/3/components/com_tags/views/tags/view.feed.php on line 54
[03-Jul-2017 18:50:01 Europe/Berlin] PHP Notice:  Undefined property: stdClass::$displayDate in /Applications/MAMP/htdocs/3/components/com_tags/views/tags/view.feed.php on line 55
[03-Jul-2017 18:50:01 Europe/Berlin] PHP Notice:  Undefined property: stdClass::$author in /Applications/MAMP/htdocs/3/components/com_tags/views/tags/view.feed.php on line 54
[03-Jul-2017 18:50:01 Europe/Berlin] PHP Notice:  Undefined property: stdClass::$displayDate in /Applications/MAMP/htdocs/3/components/com_tags/views/tags/view.feed.php on line 55
[03-Jul-2017 18:50:01 Europe/Berlin] PHP Notice:  Undefined property: stdClass::$author in /Applications/MAMP/htdocs/3/components/com_tags/views/tags/view.feed.php on line 54
[03-Jul-2017 18:50:01 Europe/Berlin] PHP Notice:  Undefined property: stdClass::$displayDate in /Applications/MAMP/htdocs/3/components/com_tags/views/tags/view.feed.php on line 55
```
### System information
3.7.3-dev (latest nightly build)
Multilanguage Site
macOS Sierra, 10.12.5
Firefox 54 (64-bit)
#### MAMP 4.1.1
- PHP 7.0.15
- MySQLi 5.6.35

**Comment by franz-wohlkoenig** [comment]
closed as having PR #16955<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16951">issues.joomla.org/tracker/joomla-cms/16951</a>.</sub>
